### PR TITLE
ROX-10003: Adjust ...Id naming to ...ID for style compliance

### PR DESCRIPTION
**Pull request by rhybrillou**
## Description

Pieces of the postgres generated store code do not comply with style requirements.
The goal of this change is to ensure generated postgres store code match style requirements.

## Checklist
- [ ] Investigated and inspected CI test results
~~- [ ] Unit test and regression tests added~~
~~- [ ] Evaluated and added CHANGELOG entry if required~~
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

## Testing Performed

CI is sufficient.

### CODE REVIEW

**Review by janisz — APPROVED** [review]

**Review by janisz — COMMENTED** [review]
Should we also change stores used in migration?

File: tools/generate-helpers/pg-table-bindings/store.go.tpl [hunk 1]
```diff
@@ -125,6 +126,38 @@ func New(db *pgxpool.Pool) Store {
     }
 }
 
+//// Used for testing
```
Comment (dashrews78) [inline tools/generate-helpers/pg-table-bindings/store.go.tpl:129]: nit:  I think the testing functions should remain at the bottom though that may just be my personal preference.

**Review by dashrews78 — APPROVED** [review]
Just one nit to consider or not consider.

File: central/activecomponent/datastore/internal/store/postgres/store.go [hunk 2]
```diff
@@ -252,6 +285,14 @@ func (s *storeImpl) copyFromActiveComponentsActiveContextsSlices(ctx context.Con
...
 
+func (s *storeImpl) acquireConn(ctx context.Context, op ops.Op, typ string) (*pgxpool.Conn, func(), error) {
+	defer metrics.SetAcquireDBConnDuration(time.Now(), op, typ)
+	conn, err := s.db.Acquire(ctx)
+	if err != nil {
+		return nil, nil, err
+	}
+	return conn, conn.Release, nil
+}
 func (s *storeImpl) copyFrom(ctx context.Context, objs ...*storage.ActiveComponent) error {
```
Comment (md2119) [inline central/activecomponent/datastore/internal/store/postgres/store.go:296]: newline

**Review by md2119 — CHANGES_REQUESTED** [review]

File: central/activecomponent/datastore/internal/store/postgres/store.go [hunk 3]
```diff
@@ -334,12 +377,104 @@ func (s *storeImpl) UpsertMany(ctx context.Context, objs []*storage.ActiveCompon
...
+		}
+
+		identifierBatch := identifiers[:localBatchSize]
+		q := search.ConjunctionQuery(
+			sacQueryFilter,
+			search.NewQueryBuilder().AddDocIDs(identifierBatch...).ProtoQuery(),
+		)
+
+		if err := postgres.RunDeleteRequestForSchema(ctx, schema, q, s.db); err != nil {
+			err = errors.Wrapf(err, "unable to delete the records.  Successfully deleted %d out of %d", numRecordsToDelete-len(identifiers), numRecordsToDelete)
```
Comment (md2119) [inline central/activecomponent/datastore/internal/store/postgres/store.go:466]: This message does not indicate the resource kind. It becomes a problem if this delete is just a steps belonging to a workflow. For example, deleting service account, namespaces, etc. when a cluster is deleted.

This message could be polished to indicate the resource kind like `unable to delete the %s records...", baseTable, ...`

File: central/activecomponent/datastore/internal/store/postgres/index.go [hunk 4]
```diff
@@ -53,7 +53,7 @@ func (b *indexerImpl) AddActiveComponents(_ []*storage.ActiveComponent) error {
 	return nil
 }
 
-func (b *indexerImpl) DeleteActiveComponent(id string) error {
```
[the remark this answers is not in the document]
Comment (rhybrillou) [reply central/activecomponent/datastore/internal/store/postgres/index.go]: Reverted the change on the index template.

**Review by md2119 — APPROVED** [review]

**Review by c-du — APPROVED** [review]

**Pull request merged**